ceph.spec.in

```diff
@@ -1827,6 +1827,8 @@ fi
 %{_bindir}/radosgw-token
 %{_bindir}/radosgw-es
 %{_bindir}/radosgw-object-expirer
+%{_bindir}/rgw-orphan-list
+%{_bindir}/ceph-diff-sorted
```

Comment:
could you apply following patch
```diff
diff --git a/debian/radosgw.install b/debian/radosgw.install
index 303791694c..b2930879ad 100644
--- a/debian/radosgw.install
+++ b/debian/radosgw.install
@@ -1,7 +1,9 @@
 lib/systemd/system/ceph-radosgw*
+usr/bin/ceph-diff-sorted
 usr/bin/radosgw
 usr/bin/radosgw-es
 usr/bin/radosgw-object-expirer
 usr/bin/radosgw-token
+usr/bin/rgw-orphan-list
 usr/lib/libradosgw.so*
 usr/share/man/man8/radosgw.8
```
as i think we also can benefit from these enhancements if they can be included in debian packages. FWIW, i am trying to list them in the alphabetical order.